### ISSUE TITLE
Doors not leaving a rift behind when blown up with TNT

### ISSUE THREAD

**Issue by Ulysses2** [DimensionalDevelopment/DimDoors#91]
Minecraft version: 1.12.2
Forge version: 14.23.4.2749
Dimensional Doors version: 3.0.9+287

Iron dimensional doors resist explosions so they aren't affected by this bug.
Here's a clip of it happening: https://gfycat.com/AccomplishedTediousDuiker

Placing a door on a blastproof block like obsidian prevents this from happening, although on occasion a rift will survive even when this isn't the case.

**Comment by Robijnvogel** [comment]
To clarify: A Rift that is already without a door, as in, a Rift Scar, the thing with the fractal render does not get blown up by TNT, does it?

**Comment by Ulysses2** [comment]
That's correct. The bare rifts are immune to explosions, they're only vulnerable when a door is placed on them, or when they spawn with a portal on them in world generation. I was going to mention that but forgot when I was rewriting the report.

**Comment by Darkmega18** [comment]
This is a massive issue in that one room with the walls that constantly close in and the entrance immediately forces tnt on you. that room is stupid with this as an issue.
also doesn't help that the monoliths that spawn in said room are incapable of limboing you as an alternative way out.
